<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MySerialPort</class>
 <widget class="QWidget" name="MySerialPort">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000000</width>
    <height>1000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MySerialPort</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>串口助手</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>示波器</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>保留</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>保留</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0" rowspan="6">
            <widget class="QTextEdit" name="recetEdit"/>
           </item>
           <item row="0" column="1" rowspan="6" colspan="3">
            <widget class="QTableWidget" name="inputTable">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>380</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="columnCount">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="2">
              <widget class="QLabel" name="databitLabel">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="text">
                <string>数据位</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="stopbitLabel">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="text">
                <string>停止位</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="parityLabel">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="text">
                <string>校验</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="baudRateLabel">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="text">
                <string>波特率</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="portNameLabel">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="text">
                <string>串口号</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLabel" name="rTypeLabel">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="text">
                <string>接收格式</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QComboBox" name="portNameBox"/>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="stopbitBox">
               <item>
                <property name="text">
                 <string>1位</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1.5位</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2位</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="6" rowspan="2">
              <widget class="QPushButton" name="connectBtn">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>未连接</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QComboBox" name="rTypeBox">
               <item>
                <property name="text">
                 <string>十六进制</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>字符串</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>文件</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="baudRateBox">
               <property name="currentIndex">
                <number>6</number>
               </property>
               <item>
                <property name="text">
                 <string>150</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>300</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>38400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>57600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>115200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>128000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>256000</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QComboBox" name="parityBox">
               <item>
                <property name="text">
                 <string>无校验</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>奇校验</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>偶校验</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>置1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>置0</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="databitBox">
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>5位</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6位</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7位</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8位</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="8" rowspan="2">
              <widget class="QPushButton" name="pushButton_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>清空</string>
               </property>
              </widget>
             </item>
             <item row="0" column="7">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="7" column="2">
            <widget class="QPushButton" name="addItemBtn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>添加</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QPushButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>循环</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>310</y>
           <width>431</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>横轴分辨率</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>最小值</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>最大值</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>数据类型</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>u8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>s8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>u16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>s16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>u32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>s32</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>double</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>float</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QCustomPlot" name="customPlot" native="true">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>691</width>
           <height>301</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
